samples: nrf9160: modem_shell: ppp: uart configuration option (#5456)
nrfconnect/sdk-nrf: samples/nrf9160/modem_shell/README.rst

.. note::

>    If using Windows, dial-up connection is non functional when using Segger virtual UART ports.

Reviewer: ```suggestion
   In Windows, the dial-up connection is not functional when using Segger virtual UART ports.
```

Author: Done

   PPP has been used successfully with FTDI UART port though.
   Refer to `nRF9160 Hardware Verification Guidelines - UART interface`_.

>    PPP has been successfully used with running Ubuntu Linux in virtualization environment hosted by Windows.

Reviewer: ```suggestion
   PPP has been successfully used running Ubuntu Linux in a virtualization environment hosted by Windows.
```

>    In that kind of environment, usage has been possible also with plain Segger UART ports.

Reviewer: ```suggestion
   In the hosted virtual Linux environment, using PPP is possible also with plain Segger UART ports.
```

Author: Done

   ppp down

> Set the custom configuration for ppp uart::

Reviewer: ```suggestion
Set the custom configuration for PPP UART::
```

Author: Done

>    ppp uartconf <options>

Reviewer 2: Use real options here as an example so remove `<options>` and add something that could work in correct setup. Similarly, sms phone number in an example is +987654321.

Author: Done.
